build-support/bin/xz/mac/10.13/5.2.4-1/build-xz.sh

```diff
@@ -22,7 +22,12 @@ function fetch_extract_xz_source_release {
 
```

Comment:
The idea is to leave copies of the scripts used to produce each binary, so that we could (attempt to) reproduce the build later. So this should either be a copy or a symlink to a shared copy, rather than a rename.

Reply:
I added back version `5.2.4` -- I chose to keep the file as it was (without `--disable-rpath`) instead of using a symlink.